### DISCUSSION TITLE
Replicate volume with multiple files vs one file of the same size?

### DISCUSSION THREAD

**Discussion by winkrs** [backube/volsync#1156] [Q&A]
Why does it take more time to replicate volume of the same size having multiple files than a volume having just one file?

I noticed that when the number of files increases, the replication duration took more time to complete compare to volume with just one file even though the size of the volumes are the same. Even the resource usage of volume with multiple files is way higher compare to the on with just one file.

**Comment by tesshuflower** [comment]
This is going to depend a lot on the mover technology being used.  But generally more files means more to compare when deciding if the file on the destination exists already and whether it needs to be copied or how much/which parts to copy if the file has changed.  For specifics it's probably best to refer to the documentation for the specific mover technology (rsync, rclone, restic, syncthing) that you're using.
